tests/common/wifi/aws_test_wifi.c

```diff
@@ -2253,14 +2345,15 @@ static void prvConnectionTask( void * pvParameters )
         {
             snprintf( pxTaskParams->cStatusMsg,
                       sizeof( pxTaskParams->cStatusMsg ),
-                      "Task %d failed to connect to the AP %s with error code %d.\r\n",
+                      "FAILURE: Task %d failed to connect to the AP %s with error code %d.\r\n",
                       pxTaskParams->usTaskId,
                       xTestNetworkParams.pcSSID,
                       xWiFiConnectStatus );
             configPRINTF( ( pxTaskParams->cStatusMsg ) );
             break;
         }
 
+        configPRINTF( ( "INFO: Task %d connected to AP %s.\r\n", pxTaskParams->usTaskId, xTestNetworkParams.pcSSID ) );
```

Comment:
do we want different levels of print statement (like OTA has) for log and INFO?

Reply:
This was taken from 1.5 release branch